src/utils/testing.tsx

```diff
@@ -194,16 +194,15 @@ export function unloadCSS(path: string) {
 // to render without errors, you can pass a `toRender` function that
 // returns an element ready to be rendered.
 
-
 interface Options {
   skipAs?: boolean
   skipSx?: boolean
 }
 
 interface BehavesAsComponent {
-  Component: React.FunctionComponent<any>,
-  systemPropArray: any[],
-  toRender?: () => React.ReactElement,
+  Component: React.FunctionComponent<any>
```

Comment:
[nit] I think this is the more common way to write this:
```suggestion
  Component: React.ComponentType<any>
```